pages/blog/2021-summary.md

```diff
@@ -0,0 +1,669 @@
+
+I personally wish we actually reach more product managers (PM) than 2%. As a person that was a PM in the past, I remember how much specs and APIs were important for products and hope other PMs go that path too.
+
+### Followers and impressions
+
+At the moment, we have 1534 followers. Unfortunately, LinkedIn memory is short and remembers only numbers one year back. I do not have access to January 2021 data. 
+
+In the case of a percentage increase, I take only numbers related to organic growth.
+
+<table>
+  <tr>
+    <td align="center">
+    </td>
+    <td align="center">
+      2020
+    </td>
+    <td align="center">
+      2021
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Followers
+    </td>
+    <td align="center">
+       611<br/>
+       57 from payed marketing
+    </td>
+    <td align="center">
+       745<br/>
+       (34% increase)
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Impressions
+    </td>
+    <td align="center">
+       80 095
+    </td>
+    <td align="center">
+       248 571<br/>
+       146 252 from payed marketing<br/>
+       (28% increase)
+    </td>
+  </tr>
+</table>
+
+We published 265 posts on LinkedIn, and with a growing number of followers, it converted into many users visiting our [website](http://asyncapi.com/).
+
+<table>
+  <tr>
+    <td align="center">
+    </td>
+    <td align="center">
+      2020
+    </td>
+    <td align="center">
+      2021
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Users
+    </td>
+    <td align="center">
+       715
+    </td>
+    <td align="center">
+       1 857<br/>
+       (160% increase)
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      New Users
+    </td>
+    <td align="center">
+       656
+    </td>
+    <td align="center">
+       1 699<br/>
+       (159% increase)
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      Sessions
+    </td>
+    <td align="center">
+       1 042
+    </td>
+    <td align="center">
+       2 511<br/>
+       (141% increase)
+    </td>
+  </tr>
+</table>
+
+### Sponsored marketing
+
+We spent $1200 on paid marketing on LinkedIn to increase awareness about AsyncAPI Hackathon and Conference. We noticed a large increase in impressions on our sponsored posts. These sponsored posts did not link to our AsyncAPI website but to the conference registration site. To summarize, $1200 converted into 763 visits to the registration website and ... wait for it ... wait for it ... 15 registrations. I have to tell you that 2021 taught me that there is nothing better than organic marketing. Better pay $1200 to contributors for their work rather than give it out to LinkedIn.
+
+I recommend everyone to maintain a relationship with the community also on LinkedIn. This social media is changing. Many folks that are not active on Twitter are active on LinkedIn, and you should address them. In the end, reposting stuff you have for Twitter to LinkedIn is not very time-consuming.
```

Comment:
```suggestion
I recommend everyone to maintain a relationship with the community also on LinkedIn. This social media is changing. Many folks that are not active on Twitter are active on LinkedIn, and you should target them. In the end, reposting stuff you have for Twitter to LinkedIn is not very time-consuming.
```

Maybe target instead of address? I don't know, sounded weird to me.